packages/expo-cli/src/credentials/api/android.ts

```diff
@@ -0,0 +1,72 @@
+import { ApiV2, User } from '@expo/xdl';
+import keyBy from 'lodash/keyBy';
+
+import { AndroidCredentials, Keystore } from '../credentials';
+import { Context } from '../context';
+
+export class AndroidApi {
+  private api: ApiV2;
+  private shouldRefetchAll: boolean = true;
+  private credentials: { [key: string]: AndroidCredentials } = {};
+
+  constructor(user: User) {
+    this.api = ApiV2.clientForUser(user);
+  }
+
+  public async fetchAll(): Promise<{ [key: string]: AndroidCredentials }> {
+    if (this.shouldRefetchAll) {
+      this.credentials = keyBy(
+        (await this.api.getAsync('credentials/android'))?.credentials || [],
+        'experienceName'
+      );
+      this.shouldRefetchAll = false;
+    }
+    return this.credentials;
+  }
+
+  public async fetchKeystore(experienceName: string): Promise<Keystore | null> {
+    await this._ensureCredntials(experienceName);
+    return this.credentials[experienceName]?.keystore;
```

Comment:
Hmmm, I think there is a type mismatch. The return type for this function is `Keystore | null` but I think in the case where the credentials for a given `experienceName` don't exist - the result will be `undefined`.